README.md

```diff
@@ -2,21 +2,54 @@
 
 ## Build site
 
-To build the website locally, clone the repo with:
+To build the website locally, clone the repo and change into the repo's directory.
 
 ```
 git clone https://github.com/blab/blotter.git
+cd blotter/
 ```
 
-Then install necessary Ruby dependencies by running `bundle install` from within the `blotter` directory.  After this, the site can be be built with:
+Install the latest version of Ruby supported by blotter using [rbenv](https://github.com/rbenv/rbenv) to work around any system-wide Ruby installations that may be incompatible.
+
+``` bash
+rbenv install 3.2.6
+```
+
+Set a local Ruby version for the blotter directory.
+
+``` bash
+rbenv local 3.2.6
+```
+
+Install bundler.
+
+``` bash
+gem install bundler
+```
```

Comment:
while trying to follow these instructions, i needed to do `rbenv rehash` before it found the correct `gem` — prior to that, it was trying to use `/usr/bin/gem` instead.

Reply:
I intentionally omitted instructions about rbenv installation, since that felt out of scope. I'm already having a different experience, as I try this again from my home computer. :-/

Do you want to add a `> [!TIP]` to the effect that one should check that `gem` is referring to the rbenv installation?

Reply:
I added a note that users should check where `gem` is and handle the situation accordingly.